src/fetch/transfer_file.py

```diff
@@ -62,8 +64,8 @@ class CSVTransfer:
 
     token_type: TokenType
     # Safe airdrop uses null address for native asset transfers
-    token_address: Optional[Address]
```

Comment:
Why are you reducing the strictness of type?

Reply:
Ok - so you didn't exactly highlight enough here for me to see what your question was about. 

I think the reason I did this here was that `CSVTransfer` is a raw type that is only every constructed out of a proper Transfer and that the types are more directly related to the actual File types. I could probably also put `Address` back here if you prefer (maybe it will remind me why I did it in the first place).

Reply:
I think I would always prefer to keep the strict typ as long as possible. It removes the probability of errors in the code.